README.md

```diff
@@ -302,6 +303,19 @@ algolia_search:
 
 Checkout the [indexing documentation](https://www.algolia.com/doc/api-client/symfony/indexing/) to learn how to send data to Algolia.
 
+#### Batching
+
+By default, calls to algolia to index or remove data are batched per 500 items. You can easily
+modify the batch size in your configuration.
+
+```yaml
+algolia_search:
+  batchSize: 250
```

Comment:
best practice is to use snake case here:

```diff
- batchSize:
+ batch_size:
```

Reply:
Indeed but currenlty it won't work. I'll see how I can add support for this. Any idea?